### ISSUE TITLE
Deprecation errors on main Entry Manager / Entries page of control panel

### ISSUE THREAD

**Issue by mcfarlandonline** [ExpressionEngine/ExpressionEngine#3913]
PHP 8.1 (also experienced errors in v8.2)
EE 7.3.14 (first saw errors in v6.4.15)
mysqlnd 8.0.30

Control Panel > Entries (.../cp/publish/edit) reveals the following errors: 

> Deprecated
> htmlentities(): Passing null to parameter #1 ($string) of type string is deprecated
> ee/ExpressionEngine/Service/Filter/Filter.php, line 224
> 
> Severity: E_DEPRECATED
> Warning
> Cannot modify header information - headers already sent by (output started at ee/legacy/core/Exceptions.php:120)
> ee/ExpressionEngine/Boot/boot.common.php, line 478
> 
> Severity: E_WARNING

There are no filters applied on this screen. Filtering by channels usually makes the filters go away, but some of the filters are buggy while the deprecation errors are showing. 

Errors do not appear in PHP 8.0 / EE 7.3.14

**Comment by intoeetive** [comment]
@mcfarlandonline this is most likely caused by a certain third-party fieldtype being shown as column. Which fieldtypes do you have that as selected as Fields in columns lists to show?

**Comment by mcfarlandonline** [comment]
@intoeetive, nothing out of the ordinary that I see... 
<img width="1288" alt="Screen Shot 2023-11-29 at 9 31 57 AM" src="https://github.com/ExpressionEngine/ExpressionEngine/assets/20441129/b940d898-c8fd-4076-8af5-669f7a0de002">

They do use Comments and the Comment Spam module, but I'm not sure that's related...

**Comment by intoeetive** [comment]
@mcfarlandonline try adding a field into column set, and then remove that field. This probably should make the error go away (although I'm not exactly sure why it does show up in first place)

**Comment by mcfarlandonline** [comment]
@intoeetive, unfortunately adding and/or removing fields into the column set has no effect. 

I did notice that there is a blank "Author" listed in the dropdown. Where its ID = 0 
<img width="300" alt="Screen Shot 2023-11-30 at 8 55 32 AM" src="https://github.com/ExpressionEngine/ExpressionEngine/assets/20441129/ecdf992c-f149-44d3-87e1-7a2b8be6bc05">

Might that be the issues, and if so, is there a way to zap that in the database?

**Comment by intoeetive** [comment]
@mcfarlandonline thank you, that was it! Please see the referenced PR for the fix.

I also suggest that you filter but that blank author and set correct author for entries found under that criteria (as not having author for entry might cause other issues as well)

**Comment by mcfarlandonline** [comment]
Yay! Thanks @intoeetive. 

I wasn't able to access the entry from the Entry Manager, as it turned out. Filtering by Authors wasn't working, and just going through pagination eventually stopped, when I encountered the page where the Authorless entry was. 

However, I was able to filter within the MySQL database and assign and author to that entry. It also had a Channel ID of 0, which was weird. Turned out to be an old entry we could just delete, and now things are working again as expected. Appreciate your help!
